html/modules/disabled/formdazzle/formdazzle.info.yml

```diff
@@ -0,0 +1,4 @@
+name: 'formdazzle - disabled'
```

Comment:
To import the config, I needed a disabled module for 'ckeditor' too.

Reply:
Added